pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -1689,6 +1698,51 @@ func (rc *rangeController) checkConsistencyRaftMuLocked(ctx context.Context) {
 		if !ok {
 			panic(errors.AssertionFailedf("replica %s not in replicaSet", replicaID))
 		}
+		rss := rs.sendStream
+		if rss == nil {
+			continue
+		}
+		// Check internal consistency of replicaSendStream.
+		func() {
```

Comment:
This func looks like a candidate to be a method in `replicaSendStream`. Then it could be used in this type's unit tests as well, if there are any.